library/aesce.c

```diff
@@ -66,9 +66,12 @@
 #   endif
 #endif
 
-#if !defined(__ARM_FEATURE_AES) || defined(MBEDTLS_ENABLE_ARM_CRYPTO_EXTENSIONS_COMPILER_FLAG)
+#include <arm_neon.h>
```

Comment:
I'm not sure what the right way to do this is, but should we test for __ARM_NEON before including (as per https://arm-software.github.io/acle/main/acle.html#arm_neonh )? E.g.:

```suggestion
#ifdef __ARM_NEON
#include <arm_neon.h>
#else
#error Target does not support NEON instructions
#endif /* __ARM_NEON */
```